openstack/compute/v2/servers/requests.go

```diff
@@ -189,6 +189,10 @@ type CreateOpts struct {
 	// ServiceClient will allow calls to be made to retrieve an image or
 	// flavor ID by name.
 	ServiceClient *gophercloud.ServiceClient `json:"-"`
+
+	// Tags allows a server to be tagged with single-word metadata.
+	// Requires microversion 2.52 or later.
+	Tags []string `json:"tags,omitempty" mv:"min:2.52"`
```

Comment:
Not a deal-breaker, but it seems odd to add every request option to the base `RequestOpts` struct. Then again, maybe with the impending versioning it makes sense.

Reply:
Yeah, the duality of requests vs responses is what's been driving me nuts for the past year.

I thought about adding a `CreateOptsExt` to `microversions.go` which would kind of bring the style more inline with the response, but I'm not a fan. One reason is because after a while, the microversion fields will be so standard that an opts extension will always be used. Another is because microversion-added fields aren't really an extension. 

Also, we've already merged a few microversion-specific request fields in other packages. When they were added, it didn't feel wrong to add them there, so this just continues with that style.

Reply:
Looking at this again, should `[]string` be changed to `Tags`? It looks like I unintentionally swapped the style here.

Reply:
No, return types should be built-in types if possible.

Reply:
> No, return types should be built-in types if possible.

Sorry, I meant in `CreateOpts`, should `Tags` be of type`Tags` even though it's really just an alias to `[]string`. If not, then is there a point to even having the `Tags` type? Act as a list of additional available fields?

Reply:
Ah, of course, this is the requests file. Still, I think it should be a built-in type since the element values are unconstrained.

Reply:
OK, so should the `Tags` type be nixed?

Reply:
I think so